content/docs/get-started/gcp/create-project.md

```diff
@@ -40,6 +40,10 @@ $ pulumi new gcp-python
 <div class="language-prologue-go"></div>
 
 ```bash
+# install the pulumi gcp plugin
+# check for the release version here https://github.com/pulumi/pulumi-gcp/releases
+$ pulumi plugin install resource gcp 2.6.0
```

Comment:
I assume we need similar in AWS, Azure, Kubernetes?

Reply:
Yup, I just made the changes in separate PRs. 
https://github.com/pulumi/docs/pull/2480
https://github.com/pulumi/docs/pull/2482
https://github.com/pulumi/docs/pull/2483
https://github.com/pulumi/templates/pull/96
https://github.com/pulumi/templates/pull/97